server/src/application/application.entity.ts

```diff
@@ -45,4 +46,7 @@ export class Application extends CustomBaseEntity {
 
   @ManyToOne(() => FormMetaData, (form) => form.applications)
   form: FormMetaData;
+
+  @ManyToOne(() => User, (user) => user.applications)
+  user: User;
```

Comment:
I think this will change to many-to-many. But not for now!

Reply:
Will leave a comment above the line, thank you!